packages/repo/src/util.ts

```diff
@@ -17,61 +17,39 @@ export const ucanForOperation = async (
   return auth.encodeUcan(ucanForOp)
 }
 
-export const getCommitPath = async (
-  blockstore: IpldStore,
-  earliest: CID | null,
-  latest: CID,
-): Promise<CID[] | null> => {
-  let curr: CID | null = latest
-  const path: CID[] = []
-  while (curr !== null) {
-    path.push(curr)
-    const commit = await blockstore.get(curr, def.commit)
-    if (earliest && curr.equals(earliest)) {
-      return path.reverse()
-    }
-    const root = await blockstore.get(commit.root, def.repoRoot)
-    if (!earliest && root.prev === null) {
-      return path.reverse()
-    }
-    curr = root.prev
-  }
-  return null
-}
-
 export const getWriteOpLog = async (
-  blockstore: IpldStore,
-  earliest: CID | null,
+  storage: RepoStorage,
   latest: CID,
+  earliest: CID | null,
 ): Promise<RecordWriteOp[][]> => {
```

Comment:
I take it this is what bonked the pds sync test 😆

Reply:
haha no it was actually the recursive sql query which returned
latest -> earliest when i expected the other way